R/showmekittens.R

```diff
@@ -23,7 +23,18 @@ showmekittens <- function(x = NULL, list = FALSE){
     pool <- c(capucine = "http://www.youtube.com/watch?v=KIePsbJSS04",
               vacuum = "https://www.youtube.com/watch?v=9J4UKYm9320",
               eclipse = "https://www.youtube.com/watch?v=RsUH2MuY33o",
-              nom     = "https://www.youtube.com/watch?v=Cl5Pfc5TyO0")
+              nom     = "https://www.youtube.com/watch?v=Cl5Pfc5TyO0"
```

Comment:
Hi Becca! This [didn't pass the automated tests](https://travis-ci.org/thibautjombart/adegenet/jobs/405457088#L4867-L4871) because there should be commas at the end of these lines. If you go back to your `patch-2` branch in your fork (https://github.com/rdecamp/adegenet/tree/patch-2), you should be able to add them in as another commit.

Reply:
Thanks for letting me know! Should be all set now, with commas AND more cat videos suggested by friends!